src/sentry/utils/suspect_resolutions/commit_correlation.py

```diff
@@ -19,10 +24,12 @@ def is_issue_commit_correlated(resolved_issue: int, candidate_issue: int, projec
     )
 
 
-def get_files_changed(issue_id: int, project_id: int) -> Set:
-    releases = GroupRelease.objects.filter(group_id=issue_id, project_id=project_id).values_list(
-        "release_id", flat=True
-    )
+def get_files_changed(issue_id: int, project_id: int) -> Tuple[List[int], Set[str]]:
+    releases = GroupRelease.objects.filter(
+        group_id=issue_id,
+        project_id=project_id,
+        last_seen__gte=(timezone.now() - timedelta(hours=5)),
```

Comment:
I'm using `timezone.now()` here since this `get_suspect_resolutions()` is called when an issue is resolved, so the resolution time of the resolved issue should be around the same time. It would be more accurate to get the resolution time for the resolved issue, but then we'd have to check whether the issue being passed to `get_files_changed()` is the resolved issue or the candidate issue... but this should help us narrow down the number of releases we're looking at for now.

Reply:
Yeah, we should look to parameterize the resolution time later on. For now, I suppose this is alright.